lib/models/instrumentation.js

```diff
@@ -227,6 +244,8 @@ class Instrumentation {
     let filename = `instrumentation.${name}`;
     if (name === 'build') {
       filename += `.${this.instrumentations.build.count}`;
+    } else if (name === 'initAddons') {
+      filename += `.${this.project.name().replace(/[^\w]/g, '')}`;
```

Comment:
🔪

Reply:
Removed - this will go in with the PR to set up addon-initialization instrumentation.